jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.css

```diff
@@ -133,3 +133,8 @@
     0%   { box-shadow: 0 0 10px 2px #5bc0de; }
     100% { box-shadow: 0 0 10px 2px transparent; }
 }
+
+/* Declarative widgets specific styles */
+.urth-dashboard .cell .urth-widget-error {
+    display: none;
+}
```

Comment:
Issue #242 says to only do this in preview, so should this be `.urth-dashboard.view-only ...`?